src/js/AppPermissions.js

```diff
@@ -5,19 +5,21 @@ import AppHeader from './containers/Header';
 import sdlController from './Controllers/SDLController';
 import { Link } from 'react-router-dom';
 import store from './store';
-import { closePermissionsView } from './actions'
+import { closePermissionsView, clearAppAwaitingPermissions } from './actions'
 
 const mapStateToProps = (state) => {
     var theme = state.theme
     var editingAppID = state.system.editingPermissionsAppId;
     var allowedFunctions = state.system.allowedFunctions;
+    var pendingActivation = state.system.permissionsAppAwaitingActivation;
 
     return {
         theme: theme,
         editingAppID: editingAppID,
         allowedFunctions: allowedFunctions,
         activeLayout: state.ui[editingAppID]?.displayLayout,
-        appOptions: state.appList
+        appOptions: state.appList,
+        pendingActivation: pendingActivation
```

Comment:
I realized after writing this that theoretically there can be multiple apps pending activation here, and this should be an array. let me know if you like this solution and if so I will change this variable to handle multiple apps.